db/migrate/20220812150721_add_email_from_identity_to_trn_requests.rb

```diff
@@ -0,0 +1,5 @@
+class AddEmailFromIdentityToTrnRequests < ActiveRecord::Migration[7.0]
+  def change
+    add_column :trn_requests, :email_from_identity, :boolean, default: false
```

Comment:
Hmm, I think it's possible that more than the email might come from Identity in the future:

```suggestion
    add_column :trn_requests, :from_get_an_identity, :boolean, default: false
```